### DISCUSSION TITLE
FileNotfoundError

### DISCUSSION THREAD

**Discussion by Vibhasprasad21** [beeware/briefcase#1998] [Q&A]
I followed the beeware tutorial macOS, facing an error with briefcase create. The support package installation for python version 3.12 does not have the python-stdlib directory.

briefcase create

Generating application template...

Installing support package...
Using support package https://briefcase-support.s3.amazonaws.com/python/3.12/macOS/Python-3.12-macOS-support.b4.tar.gz
Python-3.12-macOS-support.b4.tar.gz already downloaded
Unpacking support package... done
Copying standard library into app bundle... errored

Python-stdlib is missing in the support package which briefcase is installing for Python3.12 macOS

**Comment by freakboy3742** [comment]
If you're getting a download request for Python 3.12 b4, there's something very odd going on. You would only be getting that support package if you were using the *main* branch of the Briefcase template; but if you're doing the tutorial, you would have installed Briefcase from PyPI, so it should be using the branch corresponding to the version that was installed.

Can I ask you to delete the build directory, and then share the *full* log generated by `briefcase create`, rather than just the final error message?
